tests/sensors/test_http_sensor.py

```diff
@@ -103,6 +103,32 @@ def resp_check(resp):
         self.assertEqual(prep_request.url, received_request.url)
         self.assertTrue(prep_request.method, received_request.method)
 
+    @patch("airflow.hooks.http_hook.requests.Session.send")
+    def test_poke_context(self, mock_session_send):
+        """
+        test provide_context
+        """
+        response = requests.Response()
+        response.status_code = 200
+        mock_session_send.return_value = response
+
+        def resp_check(resp, context):
```

Comment:
I think it would be easy to make this test a bit more meaningful. We should check if certain parameters inside of the response are being set.

Reply:
Yes , but in the tests the context is empty , the mock do not provide any context...

Reply:
If we make it part of a dag, we should be able to get the execution date etc.

Reply:
done